README.md

```diff
@@ -87,3 +87,5 @@ Before submitting any change, make sure to:
   - Run `yarn format:prettier` if linting fails
 - Run `yarn build` if you change or add anything in `client/js` or `client/components`
 - `yarn dev` can be used to start The Lounge with hot module reloading
+
+If you would like to work on The Lounge in a containerized development environment, VS Code specific configuration files have been provided. If not prompted, use the `Rebuild and Reopen in Container` command to get going. More info in [Contributing](.github/CONTRIBUTING.md).
```

Comment:
s/containerized development environment, VS Code/VS Code devcontainer/

way shorter and if you don't know what a devcontainer is, there's the link to the explanation in CONTRIBUTING.md.

That's just a suggestion though, bikeshedding a bit here. It's just that containerized development environment doesn't mean anything, can be a million of things (nspawn, mkosi, docker, fancy mount points...)

Reply:
> s/containerized development environment, VS Code/VS Code devcontainer/
> 
> way shorter and if you don't know what a devcontainer is, there's the link to the explanation in CONTRIBUTING.md.
> 
> That's just a suggestion though, bikeshedding a bit here. It's just that containerized development environment doesn't mean anything, can be a million of things (nspawn, mkosi, docker, fancy mount points...)

addressed: https://github.com/thelounge/thelounge/pull/4514/commits/30c50438f4ea7b2045d30418a5ce10aed9ff8721#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5